src/lug/Math/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+﻿set(INCROOT ${PROJECT_SOURCE_DIR}/include/lug/Math)
+set(SRCROOT ${PROJECT_SOURCE_DIR}/src/lug/Math)
+
+# all source files
+set(SRC
+    ${SRCROOT}/Matrix.cpp
+    ${SRCROOT}/Vector.cpp
+)
+source_group("src" FILES ${SRC})
+
+# all header files
+set(INC
+    ${INCROOT}/Constant.hpp
+    ${INCROOT}/Constant.inl
+    ${INCROOT}/Export.hpp
+    ${INCROOT}/Geometry/Transform.hpp
+    ${INCROOT}/Geometry/Transform.inl
+    ${INCROOT}/Geometry/Trigonometry.hpp
+    ${INCROOT}/Geometry/Trigonometry.inl
+    ${INCROOT}/Matrix.hpp
+    ${INCROOT}/Matrix.inl
+    ${INCROOT}/Vector.hpp
+    ${INCROOT}/Vector.inl
+    ${INCROOT}/Quaternion.hpp
```

Comment:
Quaternion header files are not ordered correctly (should be after Matrix headers) and there are empty spaces before `${INCROOT}/Quaternion.inl`